src/replay/index.ts

```diff
@@ -420,6 +425,51 @@ export class Replayer {
     }
   }
 
+  public originalFit() {
```

Comment:
I'd propose renaming this to something with a verb on it so is resembles an action when you call it. The way it is looks like a property.
```suggestion
  public resetAutoFit() {
```